package.json

```diff
@@ -60,6 +60,7 @@
         "lint:types:src": "tsc --noEmit --jsx react",
         "lint:types:module_system": "tsc --noEmit --project ./tsconfig.module_system.json",
         "lint:style": "stylelint \"res/css/**/*.pcss\"",
+        "lint:workflows": "find .github/workflows -type f \\( -iname '*.yaml' -o -iname '*.yml' \\) | xargs -I {} sh -c 'echo \"Linting {}\"; action-validator \"{}\"'",
```

Comment:
Why not something like `action-validator .github/workflows/*.{yaml,yml}`

Reply:
That would be much easier, yeah. But, unfortunately, it only supports a single file argument. 😕

Reply:
The help and testing implies otherwise

![image](https://github.com/vector-im/element-web/assets/2403652/209bd22a-6b41-49e8-ac35-cb04c4dc0414)

Reply:
https://github.com/mpalmer/action-validator/blob/main/src/config.rs#L18 it takes a vector of paths

Reply:
I think that might not have been released? The version I installed through yarn bails when specifying more than one file.

Reply:
Ah maybe its not on yarn, when installing via cargo I can do multiple

Reply:
Want me to run it through cargo from within the yarn script? The xargs thing doesn't strike me as overly bad but I don't feel strongly either way.

Reply:
don't feel strongly either way

Reply:
As discussed in DM, given that we don't appear to be using cargo for anything else outside the CI yet and that adding both the install and run into the script might not be much nicer than the xargs thing, we'll keep the xargs variant for now. Adding multi-file support is tracked upstream in https://github.com/mpalmer/action-validator/issues/41. Once this is fixed, we can then easily simplify the yarn script later.

Reply:
@t3chguy given the above, I think this is good to land?